src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -95,6 +95,37 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     Helpers.parClone(cloneSpecs)
 
   }
+  def snapshotStream(
```

Comment:
be sure to confirm that the snapshot backs up ALL data from all organization_ids. This means you need to test on a workspace that has multiple workspaces in the same bronze tables.